packages/adapter-postgres/src/index.ts

```diff
@@ -63,6 +72,36 @@ export const Adapter = createAdapter<TableTypes, FieldTypes, ConnectionConfig, C
     }
 
     const results = await db.query(select.toString());
+    const loads = context.loads;
+
+    if (loads) {
+      const loadPromises = Object.keys(loads).map(async (k) => {
+        const load = loads[k];
+        const where: { [k: string]: any[] } = {};
+        where[load.keys[1]] = results.rows.map((r) => r[load.keys[0]]);
+
+        return {
+          k, load,
+          results: await this.select(db, load.select.where(where).context)};
+      });
+
+      const loaded = await Promise.all(loadPromises);
+      loaded.forEach((load) => {
```

Comment:
This looks insane haha.

Reply:
Yeah, a good refactor would probably be to choose better names for everything, and not use a tuple for the keys.